package.json

```diff
@@ -13,7 +15,7 @@
     "lint:css": "stylelint packages/**/src/*.js packages/**/theme/**/*-styles.js",
     "lint:js": "eslint --ext .js,.ts *.js scripts packages integration",
     "lint:types": "tsc",
-    "postinstall": "patch-package",
+    "postinstall": "patch-package && npm run build",
```

Comment:
Why generate the files post-install instead of having them on version control?

Reply:
We could actually commit them (and this is the first thing that I considered). 
Then we'd only need to make sure we add those manually for new components.

Reply:
Would it be enough to only run this when publishing components, like we do with web-types etc.? When switching to older branches you end up with a lot of untracked files otherwise, and since it's automatic, is there any benefit to having these files during development?

Reply:
> Would it be enough to only run this when publishing components, like we do with web-types etc.

Makes sense. Added this to `analyze` script for now (it doesn't quite fit there, but this is the easiest way to ensure this runs in CI before publishing without having to modify the release build and / or older branches).

Reply:
Sounds good, at least all the publish related things are grouped together that way.